src/main/java/racing/Car.java

```diff
@@ -0,0 +1,29 @@
+package racing;
+
+public class Car implements Comparable<Car>{
+    private String name;
+    private int movingRange;
+
+    public Car(String name,int movingRange) {
+        if (name.length() > 5) throw new IllegalArgumentException("이름은 5자 이내로 .");
```

Comment:
```suggestion
        if (name.length() > 5) {
                throw new IllegalArgumentException("이름은 5자 이내로 .");
        }
```
코드컨베션을 지켜주세요!

- [Google coding style](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)

Reply:
done